Modules/SubModule_Import/xASL_imp_BIDS2Legacy.m

```diff
@@ -26,6 +26,7 @@
 
 
     %% 1. Input check
+    fprintf('================================== BIDS to LEGACY CONVERSION =================================\n');
```

Comment:
`LEGACY` is our internal nomenclature, this has no meaning to the users, if you call it "ExploreASL legacy" this should be fine

Reply:
0d367c3